lib/SeqElastic.pm

```diff
@@ -151,6 +151,8 @@ sub go {
   
   my $fileSize = -s $filePath;
 
+  my $fileSize = -s $filePath;
```

Comment:
This library should be deleted; obviated by search/index; the corresponding Elasticsearch 5 libraries should also be removed from install-perl-libs.sh

Reply:
Also need to make sure we support the additional delimiter in search/index